### PR TITLE
Event cypress tests

### PR DESCRIPTION
**Pull request by mengweieric**
### Description
- cypress tests
- update snapshots

### Issues Resolved
[List any issues this PR will resolve]

### Check List
- [ ] New functionality includes testing.
  - [ ] All tests pass, including unit test, integration test and doctest
- [ ] New functionality has been documented.
  - [ ] New functionality has javadoc added
  - [ ] New functionality has user manual doc added
- [ ] Commits are signed per the DCO using --signoff

By submitting this pull request, I confirm that my contribution is made under the terms of the Apache 2.0 license.
For more information on following Developer Certificate of Origin and signing off your commits, please check [here](https://github.com/opensearch-project/OpenSearch/blob/main/CONTRIBUTING.md#developer-certificate-of-origin).

### CODE REVIEW

File: dashboards-observability/.cypress/integration/event_analytics.spec.js [hunk 1]
```diff
@@ -51,30 +31,33 @@ describe('Adding sample data and visualization', () => {
   });
 });
 
-describe('Has working breadcrumbs', () => {
```
Comment (eugenesk24) [inline dashboards-observability/.cypress/integration/event_analytics.spec.js:54]: I added this test 😆  Why did you remove it??

**Review by eugenesk24 — APPROVED** [review]
Looks good! happy Wednesday!!

File: dashboards-observability/.cypress/integration/event_analytics.spec.js (hunk 1 above)
Comment (eugenesk24) [reply dashboards-observability/.cypress/integration/event_analytics.spec.js:54]: Nope, just this one!

**Review by eugenesk24 — APPROVED** [review]

**Review by kavithacm — APPROVED** [review]

**Review by ps48 — APPROVED** [review]
